src/desktop/modules/routes/auth.ts

```diff
@@ -11,5 +11,8 @@ export default (app, proxyURL) => {
         "/api/auth/resolvePasswordSession/:code", authController.resolvePasswordSession.bind(authController),
     );
 
+    app.post(
```

Comment:
Add version number to api 
It can be **/api/user/v1/startSession**

Reply:
done